appvalidator/specs/webapps.py

```diff
@@ -95,6 +95,42 @@
     }
 }
 
+LANGUAGES_PROVIDED_OBJ = {
+    "expected_type": dict,
+    "process": lambda s: s.process_languages_provided,
+    "allowed_nodes": ["*"],
+    "not_empty": True,
+    "child_nodes": {
+        "*": {
+            "expected_type": dict,
+            "required_nodes": ["name", "version", "apps"],
```

Comment:
You're probably right, I'll double check. I'm not sure I have tests covering all scenarios on these nodes, so that would be a good occasion to add some.